src/applications/gi/components/vet-tec/VetTecAdditionalInformation.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class VetTecAdditionalInformation extends React.Component {
```

Comment:
This could probably be a functional component 

```javascript
export const VetTecAdditionalInformation = ({ institution: { facilityCode }) => (
// ...
);

VetTecAdditionalInformation.defaultProps = {
  institution: {}
};
```